### DISCUSSION TITLE
Github isn't Involving My commits in its contribution list but contributing the files in the repo

### DISCUSSION THREAD

**Discussion by AyeshaIrshad1337** [community/community#63919] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi here is ScreenShot that i have made more than two commits on my NLP repo by using VsCode :
![image](https://github.com/community/community/assets/104616632/0ba29d51-3695-452d-9878-0325a3f8292d)
but its only showing 2 contribution on my contribution graph 
![image](https://github.com/community/community/assets/104616632/69e88883-344f-44fa-8306-ba82650c3975)

**Comment by dushmanta05** [comment]
Hi Ayesha!

As per my knowledge, the reason could be:
- If you are committing to a branch other than **main**, then your contribution will not appear on the feed, so check if you're committing to a side branch. Once you merge the side branch with **main** the contribution will appear.
- If your local Git configuration has a different mail than the mail present in the remote GitHub account, then it won't show, so please do check that.
- Or could it be some issue with GitHub? Wait a few times.

Please do let me know if there are more details to your problem.
Thanks.

**Reply by AyeshaIrshad1337** [reply]
Yes thank you ... well  i was merging in the main branch ... but the issue was like yestarday i config on bitbucket so due to which i think this happen it was contributing but not counting ,,, , thank you

**Reply by dushmanta05** [reply]
You're welcome! And your problem got resolved right?

**Reply by AyeshaIrshad1337** [reply]
yes it did i just had to re config with the my email
